MANUAL_GCP_DEPLOYMENT.md

```diff
@@ -24,7 +24,6 @@ Client Key Pairs:
 
 5. On Cloud Run, click on the name of the application you want to mount your secrets as volume on to and click on "Edit & Deploy New Revision" on the top of the screen.
 
-<img width="649" alt="Screenshot_58" src="https://github.com/ai-cfia/membrane-backend/assets/9827730/b2b2f4b5-4cb6-4bf5-bc15-519c25d06204">
-
+![GCP Deploy & Edit](assets/gcp-edit-&-deploy.png)
```

Comment:
move image and markdown doc to repository docs/ instead ; assets has a meaning for web apps so we don't want to use that name for the folder.

Reply:
done